### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by marcusxavier85**
Plz review

### CODE REVIEW

File: pokedex_2020-09-28.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+# Dump of table kanto_pokedex
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `kanto_pokedex`;
+
+CREATE TABLE `kanto_pokedex` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `pokedex_no` int(11) DEFAULT NULL,
+  `name` char(255) DEFAULT NULL,
```
Comment (mporam) [inline pokedex_2020-09-28.sql]: please use varchar, not char

File: pokedex_2020-09-28.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  `type` char(255) DEFAULT NULL,
```
Comment (mporam) [inline pokedex_2020-09-28.sql]: same here

**Review by mporam — CHANGES_REQUESTED** [review]

File: pokedex_2020-09-28.sql (hunk 1 above)
Comment (marcusxavier85) [reply pokedex_2020-09-28.sql]: Done.

File: pokedex_2020-09-28.sql (hunk 2 above)
Comment (marcusxavier85) [reply pokedex_2020-09-28.sql]: Done.

**Review by mporam — APPROVED** [review]

**Pull request merged**